lib/utils/getTransactionOptions/index.ts

```diff
@@ -0,0 +1,26 @@
+import { ethers } from "ethers";
+import { urls } from "../../constants";
+
+/** Returns the fast gas price from polygonscan plus 10 percent*/
+export const getGasPrice = async () => {
+  try {
+    const response = await fetch(urls.polyscanGasTracker);
+    const data = await response.json();
+    if (data.message == "OK") {
+      const targetGasPrice = (
+        parseFloat(data.result.FastGasPrice) * 1.1
```

Comment:
> I can't test right now, could you add a before and after screenshot of the metamask gas screen so I can see the increased value? (Be careful not to doxx any of your personal 0x info)

I put the screenshots but I am not sure if it is very accurate since recommended gas price changes every 30 seconds and I cannot capture them simultaneously.